ui/app/src/components/DashboardList/DashboardList.tsx

```diff
@@ -0,0 +1,173 @@
+// Copyright 2023 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { DashboardResource } from '@perses-dev/core';
+import { Box, Stack } from '@mui/material';
+import { GridColDef, GridActionsCellItem, GridRowParams, GridValueGetterParams } from '@mui/x-data-grid';
+import DeleteIcon from 'mdi-material-ui/DeleteOutline';
+import PencilIcon from 'mdi-material-ui/Pencil';
+import { useCallback, useEffect, useMemo, useState } from 'react';
+import { dashboardDisplayName } from '@perses-dev/core/dist/utils/text';
```

Comment:
Hadn’t noticed this deep import into dist folder before, looks like it’s on main branch too, we should just export this out from the core package so it can be imported from `@perses-dev/core`

Reply:
done in https://github.com/perses/perses/pull/1096